src/dxgi/dxgi_adapter.cpp

```diff
@@ -278,6 +283,15 @@ namespace dxvk {
       else
         sharedMemory += heap.size;
     }
+
+    // Some games think we are on Intel given a lack of
+    // NVAPI or AGS/atiadlxx support.
+    // Report our device memory as shared memory,
+    // and some small amount for the carveout.
+    if (options->emulateUMA && !m_adapter->isUnifiedMemoryArchitecture()) {
+      sharedMemory = deviceMemory;
```

Comment:
Won't this be bad on low VRAM cards, like 2GB?

Reply:
i mean this is literally a GTA IV hack, whatever makes that game happy should work here. Most games seem to ignore this value anyway.